packages/select/src/Select/Select.spec.tsx

```diff
@@ -696,7 +697,7 @@ describe('packages/select', () => {
 
           await waitForElementToBeRemoved(listbox);
 
-          expect(getByTextFor(button, optionText)).toBeVisible();
```

Comment:
This was implicitly testing that the button had the text value of `optionText`. I think we should still include that assertion here